native/src/components/Modal.tsx

```diff
@@ -41,32 +42,35 @@ const Modal = ({
   title,
   children,
   scrollView = true,
-}: ModalProps): ReactElement => (
-  // View needs to stay until https://github.com/digitalfabrik/integreat-app/issues/3331 is done
-  <View>
-    <RNModal
-      visible={modalVisible}
-      onRequestClose={closeModal}
-      animationType='fade'
-      supportedOrientations={['portrait', 'landscape']}>
-      <Container>
-        <Header>
-          <HeaderBox goBack={closeModal} text={headerTitle} />
-        </Header>
-        {scrollView ? (
-          <ScrollContent contentContainerStyle={{ flexGrow: 1 }}>
-            {!!title && <Caption title={title} />}
-            {children}
-          </ScrollContent>
-        ) : (
-          <Content>
-            {!!title && <Caption title={title} />}
-            {children}
-          </Content>
-        )}
-      </Container>
-    </RNModal>
-  </View>
-)
+}: ModalProps): ReactElement => {
+  const insets = useSafeAreaInsets()
+  return (
+    // View needs to stay until https://github.com/digitalfabrik/integreat-app/issues/3331 is done
+    <View>
+      <RNModal
+        visible={modalVisible}
+        onRequestClose={closeModal}
+        animationType='fade'
+        supportedOrientations={['portrait', 'landscape']}>
+        <Container paddingTop={insets.top}>
```

Comment:
It turns out SafeAreaView has it's own space by default when you add padding top it will get doubled so you have to exclude the top.
```suggestion
        <Container paddingTop={insets.top} edges={['right', 'bottom', 'left']}>
```